### ISSUE TITLE
Update docker hub containers

### ISSUE THREAD

**Issue by anki-code** [xonsh/container#3]
Hi @AstraLuma !

I've noticed that the xonsh version in the container is old:
```python 
docker run -it --rm xonsh/xonsh xonsh --version
# xonsh/0.9.24
```

We have 0.9.27 at this time and we'll have great new version in the near future.

What we should do to update container to the 0.9.27 and to the future version?

Thanks for the docker containers!

**Comment by AstraLuma** [comment]
It should be rebuilding daily

**Comment by anki-code** [comment]
By the Actions [the last action was ran 5 months ago](https://github.com/xonsh/container/actions). 

Could you please bump the update? It will be cool to have the references in the README of how to do this.

**Comment by anki-code** [comment]
@AstraLuma oh, [here it is](https://github.com/xonsh/container/actions/workflows/poke.yml):

```
Cirrus CI Poke / poke.yml
This scheduled workflow is disabled because 
there hasn't been activity in this repository for at least 60 days. 
```

**Comment by AstraLuma** [comment]
Rude!

Maybe Cirrus cron tasks won't do that?

**Comment by AstraLuma** [comment]
To be clear: no action is necessary when xonsh releases. It'll pick it up the next rebuild. (Which should be daily.)

**Comment by anki-code** [comment]
> Which should be daily

But we still have 0.9.24... why? :)

**Comment by AstraLuma** [comment]
> > Which should be daily
> 
> But we still have 0.9.24... why? :)

> This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. 

The solution is to probably replace the use of github actions with cirrus ci scheduled tasks.

**Comment by iamjackg** [comment]
I was just looking at this, and also noticed that the last image version was pushed in December 2020.

![image](https://user-images.githubusercontent.com/19474350/170774529-4aa8d62c-79d5-456f-9d68-8f23c7d1910f.png)

**Comment by anki-code** [comment]
@iamjackg I'm using [the recipe from xonsh cheatsheet](https://github.com/anki-code/xonsh-cheatsheet#try-without-installation) to build docker container with xonsh.

**Comment by iamjackg** [comment]
Well yeah, that's simple enough to do, but an official prebuilt image is a lot more convenient, and from the comments in this issue it seems like @AstraLuma was already planning to fix it?

**Comment by AstraLuma** [comment]
Yeah, I set it all up. But apparently the scheduled task stuff isn't working.

Any xore developers can manually trigger a new one.

The real fix is mentioned above.

**Comment by iamjackg** [comment]
So should I open an issue in the core repo for somebody to trigger a new build, or should I wait for you to figure out the schedule stuff?

**Comment by AstraLuma** [comment]
huh, that's funny 
![image](https://user-images.githubusercontent.com/401438/172740418-2479bea5-9d84-4900-b1ea-d3a2192bf3d8.png)

this should have been fixed.

I guess I'll email cirrus.

**Comment by anki-code** [comment]
Still have ancient version on docker hub.
```python
docker pull xonsh/xonsh
docker run -it --rm xonsh/xonsh xonsh --version
# xonsh/0.9.24
```
Workaround from https://github.com/anki-code/xonsh-cheatsheet:
```python
# Docker with certain Python version and latest release of xonsh
docker run --rm -it python:3.9-slim /bin/bash \
 -c "pip install 'xonsh[full]' && xonsh"

# Docker with certain Python version and xonsh from the master branch
docker run --rm -it python:3.9-slim /bin/bash \
 -c "apt update && apt install -y git && pip install -U git+https://github.com/xonsh/xonsh && xonsh"

```

**Comment by AstraLuma** [comment]
Sorry, cirrus said just recreate the cron. If that didn't do it, I guess we need to bother them harder.

**Comment by AstraLuma** [comment]
The job is still running, so it's not that.

I'm guessing that master got renamed main about 10 months ago? #6 fixes that.

**Comment by AstraLuma** [comment]
This is blocked by #7

**Comment by anki-code** [comment]
Solved in #9
